apps/wappush/test/unit/wappush_test.js

```diff
@@ -236,12 +257,21 @@ suite('WAP Push', function() {
     };
 
     test('the notification is sent and populated correctly', function(done) {
-      this.sinon.spy(window, 'Notification');
+      this.sinon.spy(MockNotificationHelper, 'send');
 
       WapPushManager.onWapPushReceived(message).then(function() {
         done(function checks() {
-          sinon.assert.calledWithMatch(Notification, message.sender,
-            { body: 'http://www.mozilla.org' });
+          sinon.assert.calledWithMatch(MockNotificationHelper.send, {
+            id: 'message-title',
+            args: { title: message.sender }
+          }, {
```

Comment:
Here I would indent like proposed previously -- so that the `}` does not end up on the same indentation as `sinon`.